deploy/manual/cdk.json

```diff
@@ -0,0 +1,4 @@
+{
+  "app": "npx ts-node --prefer-ts-exts stacks.ts",
```

Comment:
I removed this so that all the cdk operations were controlled via `npm run *`. Happy for it to be put back in - but that was the rationale for deleting